src/components/StyleGuide/Filtering/Accessibility.mdx

```diff
@@ -0,0 +1,167 @@
+import { Meta } from "@storybook/blocks";
+
+import { Box } from "@chakra-ui/react";
+import Image from "../../Image/Image";
+import Link from "../../Link/Link";
+
+import imageFocusLiveFiltering from "/static/filteringStyleGuide/focusLiveFiltering.gif";
+import imageFocusKeywordSearch from "/static/filteringStyleGuide/focusKeywordSearch.gif";
+import imageFocusClearingFilters from "/static/filteringStyleGuide/focusClearingFilters.gif";
+import imageFocusPagination from "/static/filteringStyleGuide/focusPagination.gif";
+import imageFocusSorting from "/static/filteringStyleGuide/focusSorting.gif";
+import { exampleWrapperStyles } from "./Overview.mdx";
+
+<Meta title="Style Guide/Filtering/Accessibility" />
+
+# Filtering accessibility
+
+The following accessibility recommendations are applicable for real-time UI
+updates resulting from user actions. If UI updates will be applied after a full
+page refresh, then no additional accommodations need to be applied.
+
+## Table of contents
+
+- {<Link href="#aria-live" target="_self">Using aria-live</Link>}
+- {<Link href="#focus-management" target="_self">Focus management</Link>}
+  - {<Link href="#applying-search-terms" target="_self">Applying search terms</Link>}
+  - {<Link href="#applying-filters" target="_self">Applying filters</Link>}
+  - {<Link href="#clearing-filters" target="_self">Clearing filters</Link>}
+  - {<Link href="#pagination" target="_self">Pagination</Link>}
+  - {<Link href="#sorting" target="_self">Sorting</Link>}
+  - {<Link href="#modal-overlay" target="_self">Modal overlay</Link>}
+
+## Using aria-live
+
+The parent element housing the `total results` heading should include the
+`aria-live="polite"` attribute. This will ensure that real-time updates made to
+the heading text are announced to screen reader users.
+
+## Focus management
+
+### Applying search terms
+
+<Box
+  display="grid"
+  gridTemplateColumns={{ base: "1fr", md: "1fr 1fr" }}
+  gap={{ base: "xs", md: "l" }}
+>
+  <Box>
+    When a search term is applied using the `keyword search` field, focus should
+    be moved to the `total results` heading.
+  </Box>
+  <Image
+    alt="Example of full-page filtering app"
+    border="1px solid var(--nypl-colors-ui-border-default)"
+    src={imageFocusKeywordSearch}
+    {...exampleWrapperStyles}
+  />
```

Comment:
@EdwinGuzman I added animated examples for most of the focus management recs.

Reply:
This looks very good!